### DISCUSSION TITLE
Prompt queuing / prompt lists from file

### DISCUSSION THREAD

**Discussion by drjenkin** [lllyasviel/Fooocus#1144] [Ideas]
Hello, is anyone able to suggest how prompt queing functionality could be added? I'd really love to be able to point to a .txt or .csv with a different prompt on each line that I can tell the UI to run while I'm AFK. Any ideas?

**Comment by StrayerJ** [comment]
Not exactly what you want, but you can currently make a custom wildcard file that references other wild cards.   That can get you many randomized prompts

ex:
```
Prompt:  Older Woman, __MyCustomWildcard__

MyCustomWildcard.txt:
__expression__, __pose__, __photoshoot-type__, __shot-portrait-type__, __shot-angle__, (__color__ __clothing-female__:1.1), __hair-female__, __location__ __interior__ __background__ background, __time__, __camera__ 
```

And there plently of wildcard collections on Civatai.. i think https://civitai.com/models/20868/200-wildcards-nsfw-and-sfw lines up with my example

**Comment by drjenkin** [comment]
Appreciate this, cheers
